README.md

```diff
@@ -190,13 +190,19 @@ a specific user:
 sudo ip tuntap add name tap0 mode tap user $USER
 sudo ip link set tap0 up
 sudo ip addr add 192.168.69.100/24 dev tap0
+sudo ip -6 addr add fe80::100/64 dev tap0
+sudo ip -6 addr add fd80::100/64 dev tap0
+sudo ip -6 route fe80::/64 dev tap0
+sudo ip -6 route fd80::/64 dev tap0
```

Comment:
Same. This command also fails for me. Did you mean `ip -6 route add default via ...`?

Reply:
Oh, I forgot the `add` after `route`